src/main/webapp/app/exam/manage/test-runs/test-run-management.component.ts

```diff
@@ -49,13 +50,13 @@ export class TestRunManagementComponent implements OnInit {
                 this.course = this.exam.course!;
                 this.course.isAtLeastInstructor = this.accountService.isAtLeastInstructorInCourse(this.course);
                 this.examManagementService.findAllTestRunsForExam(this.course.id!, this.exam.id!).subscribe(
-                    (res) => {
+                    (res: HttpResponse<StudentExam[]>) => {
                         this.testRuns = res.body!;
                     },
-                    (err) => this.onError(err),
+                    (err: HttpErrorResponse) => onError(this.jhiAlertService, err),
```

Comment:
unifiying err and error would also be a nice change, especially here with both only 3 lines apart

Reply:
Done.